hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/handlers/EthereumTransactionHandlerTest.java

```diff
@@ -67,6 +67,7 @@
 import com.hedera.node.app.spi.workflows.PreHandleContext;
 import com.hedera.node.config.data.ContractsConfig;
 import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.swirlds.config.api.Configuration;
```

Comment:
Could there be a test (or pair of tests) for `calculateFees` both ways, new flag set and not set?

Reply:
Added the tests